docs/1.5/extensions/autolinks.md

```diff
@@ -31,9 +31,35 @@ $converter = new CommonMarkConverter([], $environment);
 echo $converter->convertToHtml('I successfully installed the https://github.com/thephpleague/commonmark project with the Autolink extension!');
 ```
 
-## `@mention`-style Autolinking
+## Prebuilt Autolink Extensions Based on MentionParser
 
-As of v1.5, [mention autolinking is now handled by a Mention Parser outside of this extension](/1.5/extensions/mention/).
+As of v1.5, this library includes additional autolink extensions that can be used to automatically link "mentions"
+like `@colinodell` to Twitter, GitHub and `#473` to GitHub issues.
+
+```php
+use League\CommonMark\Environment;
+use League\CommonMark\Extension\Autolink\AutolinkTwitterHandleExtension;
+use League\CommonMark\Extension\Autolink\AutolinkGitHubHandleExtension;
+use League\CommonMark\Extension\Autolink\AutolinkGitHubIssueExtension;
+
+$environment = Environment::createCommonMarkEnvironment();
+
+// Sample Input:  `@colinodell`
+// Sample Output: `<a href="https://www.twitter.com/colinodell">@colinodell</a>`
+$environment->addExtension(new AutolinkTwitterHandleExtension());
+
+// Sample Input:  `@colinodell`
+// Sample Output: `<a href="https://www.github.com/colinodell">@colinodell</a>`
+// Note: this extension and the above cannot be used at the same time (both based on @ mention symbol).
+$environment->addExtension(new AutolinkGitHubHandleExtension());
+
+// Sample Input:  `#473`
+// Sample Output: `<a href="https://github.com/thephpleague/commonmark/issues/473">#473</a>`
+$environment->addExtension(new AutolinkGitHubIssueExtension('thephpleague/commonmark'));
+```
+
+You can implement you own [custom mention parser](/1.5/customization/mention/) for more advanced uses cases, including
+using callbacks for integrating with internal services.
```

Comment:
I think we should revert this and stick with the convention in 1.5 and 2.0 that "autolinks" and "mentions" are two separate concepts:

 - Autolinks are things that specifically implement the [GFM autolink extension](https://github.github.com/gfm/#autolinks-extension-), which bascially makes URLs and email addresses clickable links
 - Mentions are a shortcut for creating links by specifying a certain symbol prefix plus some identifier (id, username, etc) which is then expanded into a longer link.

I originally conflated the two because naming things is hard, but ultimately they are different concepts where one is backed by the GFM spec.  I'd therefore like to keep this new convention consistent throughout the documentation, namespaces, etc.

Reply:
Done. This is now a proper `MentionExtension`